### PR TITLE
Pixel dropout

### PR DESCRIPTION
**Pull request by Dipet**
This transformation randomly replaces pixels with the described value.
As I can see, the library does not have this transform or its equivalent.
#1081 will be closed after this PR.

Example:
```python
import numpy as np
import matplotlib.pyplot as plt

import albumentations as A


img = np.full([100, 100, 3], 255, np.uint8)
drop_prob = 0.1
t1 = A.Compose([A.PixelDropout(dropout_prob=drop_prob, per_channel=True, p=1)])
t2 = A.Compose([A.PixelDropout(dropout_prob=drop_prob, per_channel=False, p=1)])
t3 = A.Compose([A.PixelDropout(dropout_prob=drop_prob, per_channel=True, drop_value=None, p=1)])
t4 = A.Compose([A.PixelDropout(dropout_prob=drop_prob, per_channel=False, drop_value=None, p=1)])
t5 = A.Compose([A.PixelDropout(dropout_prob=drop_prob, per_channel=False, drop_value=[64, 128, 255], p=1)])

plt.subplot(231, title="original")
plt.imshow(img)
plt.subplot(232, title=f"True")
plt.imshow(t1(image=img)["image"])
plt.subplot(233, title=f"False")
plt.imshow(t2(image=img)["image"])
plt.subplot(234, title=f"True None")
plt.imshow(t3(image=img)["image"])
plt.subplot(235, title=f"False None")
plt.imshow(t4(image=img)["image"])
plt.subplot(236, title=f"False [64, 128, 255]")
plt.imshow(t5(image=img)["image"])
plt.show()
```

image

### CODE REVIEW

**Review by ternaus — APPROVED** [review]

File: albumentations/augmentations/transforms.py [hunk 1]
```diff
@@ -3496,3 +3498,80 @@ def get_transform_init_args_names(self):
...
+    Image types:
+        any
+    """
+
+    def __init__(
+        self,
+        dropout_prob: float = 0.01,
+        per_channel: bool = False,
+        drop_value: typing.Optional[typing.Union[float, typing.Sequence[float]]] = 0,
+        always_apply: bool = True,
```
Comment (creafz) [inline albumentations/augmentations/transforms.py:3529]: Is it a reason to set `always_apply` to `True` by default?

Comment (Dipet) [reply albumentations/augmentations/transforms.py:3529]: oops, it is typo

**Review by creafz — APPROVED** [review]

File: albumentations/augmentations/transforms.py [hunk 2]
```diff
@@ -3496,3 +3498,80 @@ def get_transform_init_args_names(self):
             "beta_limit",
             "noise_limit",
         )
+
+
+class PixelDropout(ImageOnlyTransform):
+    """Set pixels to 0 with some probability.
```
Comment (BloodAxe) [inline albumentations/augmentations/transforms.py:3504]: Set pixels to 0 -> Set random pixels to `drop_value`

**Comment by BloodAxe** [comment]
Do we want to introduce optional parameter `mask_fill_value` to also apply dropout to masks?

**Comment by Dipet** [comment]
Work in progress. I will add `mask_fill_value` and fix int32 data type, confused with uint32

**Review by ternaus — APPROVED** [review]

**Pull request merged**